From 2407453aeea8bfeca5efd75ee1d87b85e69fc4c6 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Tue, 21 Jun 2022 13:29:51 +0300
Subject: [PATCH] [SW-91832] gaudi2: erom: add efuse init

device_efuse_init() call was missing in erom recovery
application.

This was causing issue of efuse read failure on
secured system.

Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
Change-Id: Ic5bb726f7abe4193e7bc90a944db3c60e9d4b7a6
---

diff --git a/app/ext_rom/erom_recovery.c b/app/ext_rom/erom_recovery.c
index 8d1e82e..a79666a 100644
--- a/app/ext_rom/erom_recovery.c
+++ b/app/ext_rom/erom_recovery.c
@@ -80,6 +80,7 @@
 
 static void device_init(void)
 {
+	device_init_efuse();
 	device_init_global_conf();
 	device_init_uart();
 	device_init_pci();
